platform.md

```diff
@@ -276,10 +275,72 @@ All lifecycle phases:
 - MUST read `CNB_PLATFORM_API` from the execution environment and evaluate compatibility before attempting to parse other inputs (see [Platform API Compatibility](#platform-api-compatibility))
 - MUST give command line inputs precedence over other inputs
 
+#### `analyzer`
+Usage:
+```
+/cnb/lifecycle/analyzer \
+  [-analyzed <analyzed>] \
+  [-daemon] \ # sets <daemon>
+  [-gid <gid>] \
+  [-group <group>] \
+  [-layers <layers>] \
+  [-log-level <log-level>] \
+  [-previous-image <previous-image> ] \
+  [-run-image <run-image> ] \
+  [-stack-id <stack-id> ] \
+  [-stack <stack> ] \
+  [-tag <tag>...] \
+  [-uid <uid>] \
+  <image>
+```
+
+##### Inputs
+| Input             | Environment Variable  | Default Value            | Description
+|-------------------|-----------------------|--------------------------|----------------------
+| `<analyzed>`      | `CNB_ANALYZED_PATH`   | `<layers>/analyzed.toml` | Path to output analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml)
+| `<daemon>`        | `CNB_USE_DAEMON`      | `false`                  | Analyze image from docker daemon
+| `<gid>`           | `CNB_GROUP_ID`        |                          | Primary GID of the stack `User`
+| `<group>`         | `CNB_GROUP_PATH`      | `<layers>/group.toml`    | Path to group definition (see [`group.toml`](#grouptoml-toml))
+| `<layers>`        | `CNB_LAYERS_DIR`      | `/layers`                | Path to layers directory
```

Comment:
```suggestion
| `<image>`           |                            |                     | Tag reference to which the app image will be written
| `<layers>`        | `CNB_LAYERS_DIR`      | `/layers`                | Path to layers directory
```

Reply:
Added in 6480d2e975ba81d5b61d681f0177539408c834b8